test/powershell/Modules/Microsoft.PowerShell.Core/Pester.Commands.Cmdlets.GetCommand.Tests.ps1

```diff
@@ -49,4 +49,52 @@ Describe "Tests Get-Command with relative paths and wildcards" -Tag "CI" {
         Pop-Location
     }
 
+    It "Get-Command -ShowCommandInfo property field test" {
+        $commandInfo = Get-Command Get-Date -ShowCommandInfo
+        $properties = ($commandInfo | Get-Member -MemberType NoteProperty)
```

Comment:
I suggest the following, so that we will see what is not matched in the output. 
```Powershell
It 'Get-Command -ShowCommandInfo property field test' {
    $commandInfo = Get-Command Get-Date -ShowCommandInfo
    $properties = ($commandInfo | Get-Member -MemberType NoteProperty)
    $propertiesAsString =  $properties.name | out-string 
    $propertiesAsString | Should MatchExactly 'CommandType' 
    $propertiesAsString | Should MatchExactly 'Definition' 
    $propertiesAsString | Should MatchExactly 'Module' 
    $propertiesAsString | Should MatchExactly 'ModuleName' 
    $propertiesAsString | Should MatchExactly 'Name' 
    $propertiesAsString | Should MatchExactly 'ParameterSets' 
}
```

Reply:
resolved